### PR TITLE
Patterns Assembler: Create blank canvas cta

### PR DESCRIPTION
**Pull request by bangank36**
#### Proposed Changes

- Create a Blank Canvas CTA in UDP
- Create Build your own Layout step in stepper flows

#### Testing Instructions

- Visit http://calypso.localhost:3000/setup/designSetup?siteSlug={site_slug}&flags=signup/design-picker-pattern-assembler
- Confirm that the Blank Canvas CTA is in in the UDP
- Click on Get Started button should navigate to new step 'buildYourOwnLayout'
- 'buildYourOwnLayout' step is empty by far, click Back button should go back to DesignSetup step

https://user-images.githubusercontent.com/10071857/186340773-e59c1dc2-2273-4e72-aee0-dd06e90b1de7.mp4

## Reference

Related to #66785

### CODE REVIEW

File: config/development.json [hunk 1]
```diff
@@ -153,6 +153,7 @@
 		"signup/design-picker-premium-themes-checkout": true,
 		"signup/design-picker-unified": true,
 		"signup/design-picker-use-featured-picks-buttons": true,
+		"signup/build-your-own": true,
```
Comment (taipeicoder) [inline config/development.json]: @bangank36 can we "prefix" this flag with the text `design-picker` (`signup/design-picker-build-your-own`)?  And also, this section is sorted alphabetically!

**Comment by miksansegundo** [comment]
@taipeicoder Is it okay to enable the feature flag in the [config file](https://github.com/Automattic/wp-calypso/blob/trunk/config/wpcalypso.json) for testing in the environment wp-calypso?

Otherwise, the Calypso Live link is not working.

Edited: Nevermind, we can add the flag in the URL /setup/designSetup?siteSlug={site slug}&flags=signup/design-picker-build-your-own

Edited: @bangank36 updated the description in case anyone need it

File: client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx [hunk 2]
```diff
@@ -372,6 +373,31 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
...
+				<img
+					className="blank-canvas-cta__image"
+					src={ blankCanvasImageUrl }
+					alt="Blank Canvas Header"
+				/>
+			</div>
+			<h3 className="blank-canvas-cta__title"> { translate( 'Start with a blank canvas' ) } </h3>
+			<p className="blank-canvas-cta__subtitle">
+				{ translate(
+					"Can't find something you like? Create something of your own by mixing and matching patterns"
```
Comment (miksansegundo) [inline client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx]: The period at the end of this sentence is missing: `Create something of your own by mixing and matching patterns.`

File: client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx [hunk 3]
```diff
@@ -372,6 +373,31 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 		/>
 	);
 
+	const blankCanvasContent = isEnabled( 'signup/design-picker-build-your-own' ) ? (
```
Comment (miksansegundo) [inline client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx]: To be consistent with the className `blank-canvas-cta-wrapper`, we can use the name `blankCanvasCta` instead of `blankCanvasContent`.

File: client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx [hunk 4]
```diff
@@ -386,6 +412,7 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 			onCheckout={ goToCheckout }
 			premiumBadge={ <PremiumBadge isPremiumThemeAvailable={ isPremiumThemeAvailable } /> }
 			heading={ heading }
+			blankCanvas={ blankCanvasContent }
```
Comment (miksansegundo) [inline client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx]: We could avoid renaming variables when passing them for more clarity, use `blankCanvasContent={ blankCanvasContent }` or `blankCanvasCta={ blankCanvasCta }`.

File: client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx [hunk 5]
```diff
@@ -372,6 +373,31 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
...
+			</div>
+			<h3 className="blank-canvas-cta__title"> { translate( 'Start with a blank canvas' ) } </h3>
+			<p className="blank-canvas-cta__subtitle">
+				{ translate(
+					"Can't find something you like? Create something of your own by mixing and matching patterns"
+				) }
+			</p>
+			<Button
+				className="blank-canvas-cta__button"
+				onClick={ () => goToStep?.( 'buildYourOwnLayout' ) }
```
Comment (miksansegundo) [inline client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx]: We are using `buildYourOwnLayout` for the name of the step, and `signup/design-picker-build-your-own` for the flag name.

For more clarity, we could use the className `build-your-own-cta` instead of `blank-canvas-cta` because **blank-canvas** is not connected to the flag or the step name.

File: packages/design-picker/src/components/unified-design-picker.tsx [hunk 6]
```diff
@@ -517,6 +523,7 @@ const UnifiedDesignPicker: React.FC< UnifiedDesignPickerProps > = ( {
 				onPreview={ onPreview }
 				onUpgrade={ onUpgrade }
 				designs={ staticDesigns }
+				blankCanvas={ blankCanvas }
```
Comment (miksansegundo) [inline packages/design-picker/src/components/unified-design-picker.tsx]: I think the prop name `blankCanvas` is confusing here because we are already using the variable `isBlankCanvas` in the same file. 

We could use the prop name `buildYourOwnCta`, which would make more sense because we use the name **Build your own** in the step and the flag name.

**Review by miksansegundo — COMMENTED** [review]
I have found a small typo. 
For more clarity, I suggested a different name for the classNames and the prop name. I think would be a good improvement. Let me know if that makes sense.

File: client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx (hunk 2 above)
Comment (bangank36) [reply client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx]: > For more clarity, I suggested a different name for the classNames and the prop name. I think would be a good improvement. Let me know if that makes sense.

Those are great suggestion, I addressed the feedback and update accordingly

File: client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx [hunk 7]
```diff
@@ -372,6 +373,31 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 		/>
 	);
 
+	const buildYourOwnCta = isEnabled( 'signup/design-picker-build-your-own' ) ? (
```
Comment (taipeicoder) [inline client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx]: [nit] I think we can use `&&` to short-circuit this logic, since we don't really need the falsy condition.

File: packages/design-picker/src/components/unified-design-picker.tsx [hunk 8]
```diff
@@ -396,25 +399,27 @@ const StaticDesignPicker: React.FC< StaticDesignPickerProps > = ( {
...
-						isPremiumThemeAvailable={ isPremiumThemeAvailable }
-						previewOnly={ previewOnly }
-						hasDesignOptionHeader={ hasDesignOptionHeader }
-						onCheckout={ onCheckout }
-						verticalId={ verticalId }
-						hasPurchasedTheme={ wasThemePurchased( purchasedThemes, design ) }
-					/>
+				{ filteredDesigns.map( ( design, index ) => (
+					<Fragment key={ design.slug }>
+						{ index === 3 ? buildYourOwnCta : null }
```
Comment (taipeicoder) [inline packages/design-picker/src/components/unified-design-picker.tsx]: [nit] Same here, we can just short-circuit this.

Comment (taipeicoder) [reply packages/design-picker/src/components/unified-design-picker.tsx]: The CTA won't show if there are less than 3 filtered results 😢
Screen Shot 2022-08-24 at 4 50 58 PM

Comment (bangank36) [reply packages/design-picker/src/components/unified-design-picker.tsx]: I applied a fixed, but it seems the `100vh` trick to prevent page jump has compromised the position :(

Comment (taipeicoder) [reply packages/design-picker/src/components/unified-design-picker.tsx]: I just created an issue for that https://github.com/Automattic/wp-calypso/issues/66905

Comment (bangank36) [reply packages/design-picker/src/components/unified-design-picker.tsx]: So we can ignore the glitch on this PR

**Comment by bangank36** [comment]
Resolved conflict when #66901 merged

File: client/landing/stepper/declarative-flow/internals/steps-repository/build-your-own-layout/index.tsx [hunk 9]
```diff
@@ -0,0 +1,22 @@
+import { StepContainer } from 'calypso/../packages/onboarding/src';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import type { Step } from '../../types';
+
+const BuildYourOwnLayout: Step = function BuildYourOwnLayout( { navigation } ) {
```
Comment (arthur791004) [inline client/landing/stepper/declarative-flow/internals/steps-repository/build-your-own-layout/index.tsx]: We could just arrow function

```js
const BuildYourOwnLayout: Step = ( { navigation } ) => {
  ...
}
```

File: client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss [hunk 10]
```diff
@@ -187,6 +187,28 @@ $design-button-primary-color: rgb( 17, 122, 201 );
 		}
 	}
 
+	/**
+	 * Blank Canvas Button
+	 */
+	.build-your-own-cta-wrapper {
```
Comment (arthur791004) [inline client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss]: It should have background-color: `#f6f7f7`. See 7l53h5fxAUNjVRBQ82YFwn-fi-242%3A43264

File: client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx [hunk 11]
```diff
@@ -371,6 +372,31 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
...
+				<img
+					className="build-your-own-cta__image"
+					src={ blankCanvasImageUrl }
+					alt="Blank Canvas Header"
+				/>
+			</div>
+			<h3 className="build-your-own-cta__title"> { translate( 'Start with a blank canvas' ) } </h3>
+			<p className="build-your-own-cta__subtitle">
+				{ translate(
+					"Can't find something you like? Create something of your own by mixing and matching patterns."
```
Comment (arthur791004) [inline client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx]: Its color is gray-60, `#50575e`

File: client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx [hunk 12]
```diff
@@ -14,6 +14,7 @@ import { useSelect, useDispatch } from '@wordpress/data';
 import { useTranslate } from 'i18n-calypso';
 import { useRef, useState, useEffect } from 'react';
 import { useDispatch as useReduxDispatch, useSelector } from 'react-redux';
+import blankCanvasImageUrl from 'calypso/assets/images/onboarding/blank-canvas.svg';
```
Comment (arthur791004) [inline client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx]: The background of this image looks not transparent. Could you export again?

BTW, I'm okay with both svg or png but the format looks weird. If we use `svg`, it would be a file rather than a picture directly.

File: client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss [hunk 13]
```diff
@@ -187,6 +187,28 @@ $design-button-primary-color: rgb( 17, 122, 201 );
...
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+		align-items: center;
+		padding: 64px;
+		grid-column: 1 / span 3;
+
+		.build-your-own-cta_title {
```
Comment (arthur791004) [inline client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss]: I thought the styles don't load correctly. It should be
```diff
-build-your-own-cta_title {
+build-your-own-cta__title {
```

 I only saw the screenshot below

image

File: client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss [hunk 14]
```diff
@@ -187,6 +187,28 @@ $design-button-primary-color: rgb( 17, 122, 201 );
...
+			margin-top: 32px;
+			margin-bottom: 8px;
+		}
+
+		.build-your-own-cta__button {
```
Comment (arthur791004) [inline client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss]: We have to set the border-radius to 4px

File: client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss [hunk 15]
```diff
@@ -187,6 +187,28 @@ $design-button-primary-color: rgb( 17, 122, 201 );
...
+			margin-top: 32px;
+			padding: 10px 80px;
```
Comment (arthur791004) [inline client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss]: I think maybe setting the `min-width` might be better. If we set the horizontal padding and the text is longer after the translation, the width of the button would be longer

File: client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx [hunk 16]
```diff
@@ -371,6 +372,31 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 		/>
 	);
 
+	const buildYourOwnCta = isEnabled( 'signup/design-picker-build-your-own' ) && (
```
Comment (arthur791004) [inline client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx]: I think we can create a new file for it and import it into the UDP directly rather than props. And if the selected theme is a blank canvas, we just go to the `buildYourOwnLayout` step. What do you think?

File: packages/design-picker/src/components/unified-design-picker.tsx [hunk 17]
```diff
@@ -399,24 +402,28 @@ const StaticDesignPicker: React.FC< StaticDesignPickerProps > = ( {
...
+							hideFullScreenPreview={ false }
+							hideDesignTitle={ false }
+							isPremiumThemeAvailable={ isPremiumThemeAvailable }
+							previewOnly={ previewOnly }
+							hasDesignOptionHeader={ hasDesignOptionHeader }
+							onCheckout={ onCheckout }
+							verticalId={ verticalId }
+							hasPurchasedTheme={ wasThemePurchased( purchasedThemes, design ) }
+						/>
+						{ ( ( filteredDesigns.length < 3 && index === filteredDesigns.length - 1 ) ||
```
Comment (arthur791004) [inline packages/design-picker/src/components/unified-design-picker.tsx]: Is it possible to insert the blank canvas design into `filteredDesigns`?

File: config/development.json [hunk 18]
```diff
@@ -148,6 +148,7 @@
 		"sign-in-with-apple/redirect": true,
 		"signup/add-ons": false,
 		"signup/anchor-fm": true,
+		"signup/design-picker-build-your-own": true,
```
Comment (arthur791004) [inline config/development.json]: Could you rename it to `signup/design-picker-patterns-assembler`? It might be easier to understand what's this

File: client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx (hunk 12 above)
Comment (bangank36) [reply client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx]: Sure let me try again

File: packages/design-picker/src/components/unified-design-picker.tsx (hunk 17 above)
Comment (bangank36) [reply packages/design-picker/src/components/unified-design-picker.tsx]: It's good idea but I need some time for it and https://github.com/Automattic/wp-calypso/pull/66895#discussion_r954482596

Comment (bangank36) [reply packages/design-picker/src/components/unified-design-picker.tsx:423]: It's good idea but I need some time for it and https://github.com/Automattic/wp-calypso/pull/66895#discussion_r954482596

Comment (bangank36) [reply packages/design-picker/src/components/unified-design-picker.tsx]: @arthur791004 can a blank canvas design be expected from the API, as I can see the \`isBlankCanvasDesign\` is used [here](https://github.com/Automattic/wp-calypso/blob/22f0182bc9427fda885cf17e3781dc18ee220754/packages/design-picker/src/components/unified-design-picker.tsx#L304).

File: client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx (hunk 16 above)
Comment (bangank36) [reply client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx]: @arthur791004 I tried to create new file for this but can't import the image in `client/assets/onboarding/blank-canvas.png` could you advice?

https://github.com/Automattic/wp-calypso/blob/ee7456e8e53e807bd0ddbabaefee8ae85883f53f/packages/design-picker/src/components/pattern-assembler-cta/index.tsx#L16

File: packages/design-picker/src/components/unified-design-picker.tsx (hunk 17 above)
Comment (bangank36) [reply packages/design-picker/src/components/unified-design-picker.tsx]: @arthur791004 I created a blank canvas design and insert to the static designs list, could you have a look at the implementation? Thanks

**Comment by fushar** [comment]
@arthur791004 @bangank36 @taipeicoder Can we consistently call this feature "pattern assembler" as well in the code? It is a bit confusing because now we have two terms. Proposal:

- feature flag: `signup/design-picker-pattern-assembler`
- step: `patternAssembler`
- `PatternAssemblerCta`

What do you think?

(small grammar note: it is "pattern assembler", "pattern selector" etc, not "patterns assembler" / "patterns selector") 😄

**Comment by bangank36** [comment]
> @arthur791004 @bangank36 @taipeicoder Can we consistently call this feature "pattern assembler" as well in the code? It is a bit confusing because now we have two terms. Proposal:
> 
> *   feature flag: `signup/design-picker-pattern-assembler`
> *   step: `patternAssembler`
> *   `PatternAssemblerCta`
> 
> What do you think?
> 
> (small grammar note: it is "pattern assembler", "pattern selector" etc, not "patterns assembler" / "patterns selector") 😄

I updated the name for consistency cc: @miksansegundo

File: client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx [hunk 19]
```diff
@@ -383,6 +387,7 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 			onPreview={ previewDesign }
 			onUpgrade={ upgradePlan }
 			onCheckout={ goToCheckout }
+			onPatternAssemblerCtaClick={ goTopatternAssembler }
```
Comment (arthur791004) [inline client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx]: I think we don't need this. We could handle the navigation in `client/landing/stepper/declarative-flow/site-setup-flow.ts` on line 177.
```js
case 'designSetup':
	if ( selectedDesign.recipe?.stylesheet === 'blank-canvas-blocks' ) {
		return navigate( 'patternAssembler' );
	}

	return navigate( 'processing' );
```

File: packages/design-picker/src/components/unified-design-picker.tsx [hunk 20]
```diff
@@ -399,25 +410,29 @@ const StaticDesignPicker: React.FC< StaticDesignPickerProps > = ( {
...
+							hideDesignTitle={ false }
+							isPremiumThemeAvailable={ isPremiumThemeAvailable }
+							previewOnly={ previewOnly }
+							hasDesignOptionHeader={ hasDesignOptionHeader }
+							onCheckout={ onCheckout }
+							verticalId={ verticalId }
+							hasPurchasedTheme={ wasThemePurchased( purchasedThemes, design ) }
+						/>
+					) : (
+						<PatternAssemblerCta key={ design.slug } onButtonClick={ onPatternAssemblerCtaClick } />
```
Comment (arthur791004) [inline packages/design-picker/src/components/unified-design-picker.tsx]: We could just pass `onSelect` function to it

File: packages/design-picker/src/components/pattern-assembler-cta/style.scss [hunk 21]
```diff
@@ -0,0 +1,26 @@
+.pattern-assembler-cta-wrapper {
```
Comment (arthur791004) [inline packages/design-picker/src/components/pattern-assembler-cta/style.scss:4]: We have to hide it on mobile until we have the mobile version of the Pattern Assembler

File: client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx (hunk 16 above)
Comment (bangank36) [reply client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx]: Use `/calypso/images/onboarding/blank-canvas.png` for the image

File: packages/design-picker/src/components/unified-design-picker.tsx [hunk 22]
```diff
@@ -381,12 +382,19 @@ const StaticDesignPicker: React.FC< StaticDesignPickerProps > = ( {
 	purchasedThemes,
 } ) => {
 	const hasCategories = !! categorization?.categories.length;
+	const blankCanvasDesign = {
```
Comment (arthur791004) [inline packages/design-picker/src/components/unified-design-picker.tsx]: Could we keep it the same as real data? thanks!

image

Comment (bangank36) [reply packages/design-picker/src/components/unified-design-picker.tsx]: Sure, where do you find it?

Comment (arthur791004) [reply packages/design-picker/src/components/unified-design-picker.tsx]: I just modify the endpoint to let it return the blank canvas! I think the most important part is `slug` and `recipe.stylesheet`. We can ignore the categories

File: packages/design-picker/src/components/unified-design-picker.tsx [hunk 23]
```diff
@@ -381,12 +382,19 @@ const StaticDesignPicker: React.FC< StaticDesignPickerProps > = ( {
...
+		title: 'Blank Canvas',
+	} as Design;
+
 	const filteredDesigns = useMemo( () => {
 		const result = categorization?.selection
 			? filterDesignsByCategory( designs, categorization.selection )
 			: designs.slice(); // cloning because otherwise .sort() would mutate the original prop
 
 		result.sort( sortDesigns );
+		result.splice( result.length < 3 ? result.length : 3, 0, blankCanvasDesign );
```
Comment (fushar) [inline packages/design-picker/src/components/unified-design-picker.tsx]: [nit]

```suggestion
		result.splice( Math.min(result.length, 3), 0, blankCanvasDesign );
```

File: packages/design-picker/src/components/unified-design-picker.tsx [hunk 24]
```diff
@@ -399,25 +407,29 @@ const StaticDesignPicker: React.FC< StaticDesignPickerProps > = ( {
...
-						isPremiumThemeAvailable={ isPremiumThemeAvailable }
-						previewOnly={ previewOnly }
-						hasDesignOptionHeader={ hasDesignOptionHeader }
-						onCheckout={ onCheckout }
-						verticalId={ verticalId }
-						hasPurchasedTheme={ wasThemePurchased( purchasedThemes, design ) }
-					/>
-				) ) }
+				{ filteredDesigns.map( ( design ) =>
+					! isBlankCanvasDesign( design ) ? (
```
Comment (fushar) [inline packages/design-picker/src/components/unified-design-picker.tsx]: We already have a bunch of `isBlankCanvasDesign()` logic already inside the `DesignButtonContainer` component. So perhaps we can render the `PatternAssemblerCta` there as well, to avoid duplication here?

Comment (bangank36) [reply packages/design-picker/src/components/unified-design-picker.tsx]: Good idea

File: packages/design-picker/src/components/unified-design-picker.tsx (hunk 22 above)
Comment (bangank36) [reply packages/design-picker/src/components/unified-design-picker.tsx]: Got it

File: packages/design-picker/src/components/unified-design-picker.tsx [hunk 25]
```diff
@@ -381,12 +387,22 @@ const StaticDesignPicker: React.FC< StaticDesignPickerProps > = ( {
...
+		title: 'Blank Canvas',
+	} as Design;
+
 	const filteredDesigns = useMemo( () => {
 		const result = categorization?.selection
 			? filterDesignsByCategory( designs, categorization.selection )
 			: designs.slice(); // cloning because otherwise .sort() would mutate the original prop
 
 		result.sort( sortDesigns );
+		result.splice( Math.min( result.length, 3 ), 0, blankCanvasDesign );
```
Comment (arthur791004) [inline packages/design-picker/src/components/unified-design-picker.tsx]: We need to check feature flag `design-picker-pattern-assembler` here or in `packages/design-picker/src/components/unified-design-picker.tsx`

File: packages/design-picker/src/components/unified-design-picker.tsx [hunk 26]
```diff
@@ -319,6 +320,11 @@ const DesignButtonContainer: React.FC< DesignButtonContainerProps > = ( {
 				disabled={ ! isBlankCanvas && ! previewOnly }
```
Comment (fushar) [inline packages/design-picker/src/components/unified-design-picker.tsx]: We can remove `! isBlankCanvas` here too as it is already checked above.

File: packages/design-picker/src/components/pattern-assembler-cta/index.tsx [hunk 27]
```diff
@@ -0,0 +1,38 @@
...
+			<div className="pattern-assembler-cta__image-wrapper">
+				<img
+					className="pattern-assembler-cta__image"
+					src="/calypso/images/onboarding/blank-canvas.png"
+					alt="Blank Canvas Header"
+				/>
+			</div>
+			<h3 className="pattern-assembler-cta__title">
+				{ ' ' }
+				{ translate( 'Start with a blank canvas' ) }{ ' ' }
```
Comment (arthur791004) [inline packages/design-picker/src/components/pattern-assembler-cta/index.tsx]: I think we don't need `{ ' ' }` before and after

**Review by arthur791004 — APPROVED** [review]
LGTM 👍

But as discussed here: 7l53h5fxAUNjVRBQ82YFwn-fi-242%3A43264, could you create a follow-up PR to only show the blank canvas under the “All” tab? thanks!

File: packages/design-picker/src/components/unified-design-picker.tsx [hunk 28]
```diff
@@ -381,12 +387,23 @@ const StaticDesignPicker: React.FC< StaticDesignPickerProps > = ( {
...
+		title: 'Blank Canvas',
+	} as Design;
+
 	const filteredDesigns = useMemo( () => {
 		const result = categorization?.selection
 			? filterDesignsByCategory( designs, categorization.selection )
 			: designs.slice(); // cloning because otherwise .sort() would mutate the original prop
 
 		result.sort( sortDesigns );
+		isEnabled( 'signup/design-picker-pattern-assembler' ) &&
```
Comment (fushar) [inline packages/design-picker/src/components/unified-design-picker.tsx]: I found this a bit hard to read; can we just use a simple `if` statement instead? Also, we can move the `blankCanvasDesign` declaration inside this `if`, because it's only used there.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7502836

Hi @bangank36, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include all of the following strings:
- `Start with a blank canvas`
- `Can't find something you like? Create something of your own by mixing and matching patterns.`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7502836).
